src/facade/dragonfly_connection.cc

```diff
@@ -277,28 +277,6 @@ string_view Connection::PubMessage::Message() const {
   return {buf.get() + channel_len, message_len};
 }
 
-struct Connection::DispatchOperations {
```

Comment:
I just moved it down because it was in an almost random place and all the definitions are actually 100 lines below